File: VRSampleApp/app/src/main/java/com/ooyala/sample/parser/AdType.java
package com.ooyala.sample.parser;

import com.google.gson.annotations.SerializedName;

public enum AdType {
  @SerializedName("NO-ADS")
  NOADS,

  @SerializedName("OOYALA")
  OOYALA,

  @SerializedName("IMA")
  IMA,

  @SerializedName("VAST")
  VAST
}

File: VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/parser/AdType.kt
package com.ooyala.sample.parser

import com.google.gson.annotations.SerializedName

enum class AdType {

  @SerializedName("NO-ADS")
  NOADS,

  @SerializedName("OOYALA")
  OOYALA,

  @SerializedName("IMA")
  IMA,

  @SerializedName("VAST")
  VAST
}